frontend/src/pages/Afterward/index.js

```diff
@@ -26,28 +28,41 @@ type Props = {
   afterwardMessage: String,
 };
 
+const clearSessionStorage = async () => await sessionStorage.clear();
+
 class Afterward extends React.Component<Props> {
+  state = {
+    message: '',
+  };
+
   componentDidMount() {
-    sessionStorage.clear();
-    persistor.purge();
-    clearInterval();
+    this.setState(
+      {
+        message: this.props.afterwardMessage,
+      },
+      () => {
+        getRange(0, 100).map(i => clearInterval(i)); //bad habits
```

Comment:
Can we use `clearInterval()` with the ID that was set for the clock?
If we saved the timer's reference in session storage when it's initialised with `setInterval` then we could just cancel that on instead of looping through a set.